test/fixtures/integration-test/netlify/edge-handlers/hello_world.js

```diff
@@ -1,13 +1,11 @@
-// eslint-disable-next-line you-dont-need-lodash-underscore/concat
-import { concat } from 'lodash'
```

Comment:
Unsure if this was intentional for testing purposes. If so I can revert the changes.